anomaly_detection/bayesian_changepoints.md

```diff
@@ -1,3 +1,4 @@
+<script src="../dato/js/recview.js"></script>
```

Comment:
There is a way to include this, but I felt that including it on *all* pages might be confusing to the user. Some userguide pages already serve the purpose of trying to surface relevant content, so I opted to not show the recommender on those pages.